__tests__/GithubUtils-test.js

```diff
@@ -80,6 +85,161 @@ describe('GithubUtils', () => {
+
+        test('Test no verified PRs', () => {
+            githubUtils.generateStagingDeployCashBody(tag, basePRList)
+                .then((issueBody) => {
+                    // eslint-disable-next-line max-len
+                    expect(issueBody).toBe(`${baseExpectedOutput}${openCheckbox} ${basePRList[3]}\r\n${openCheckbox} ${basePRList[0]}\r\n${openCheckbox} ${basePRList[1]}\r\n`);
+                });
+        });
+
+        test('Test some verified PRs', () => {
+            githubUtils.generateStagingDeployCashBody(tag, basePRList, [basePRList[0]])
+                .then((issueBody) => {
+                    // eslint-disable-next-line max-len
+                    expect(issueBody).toBe(`${baseExpectedOutput}${openCheckbox} ${basePRList[3]}\r\n${closedCheckbox} ${basePRList[0]}\r\n${openCheckbox} ${basePRList[1]}\r\n`);
+                });
+        });
+
+        // eslint-disable-next-line max-len
+        const allVerifiedExpectedOutput = `${baseExpectedOutput}${closedCheckbox} ${basePRList[3]}\r\n${closedCheckbox} ${basePRList[0]}\r\n${closedCheckbox} ${basePRList[1]}\r\n`;
+        test('Test all verified PRs', () => {
+            githubUtils.generateStagingDeployCashBody(tag, basePRList, basePRList)
+                .then((issueBody) => {
+                    expect(issueBody).toBe(allVerifiedExpectedOutput);
+                });
+        });
+
+        const deployBlockerHeader = '\r\n**Deploy Blockers:**\r\n';
+        test('Test no resolved deploy blockers', () => {
+            githubUtils.generateStagingDeployCashBody(tag, basePRList, basePRList, baseDeployBlockerList)
+                .then((issueBody) => {
+                    // eslint-disable-next-line max-len
+                    expect(issueBody).toBe(`${allVerifiedExpectedOutput}${deployBlockerHeader}${openCheckbox} ${baseDeployBlockerList[0]}\r\n${openCheckbox} ${baseDeployBlockerList[1]}\r\n`);
+                });
+        });
+
+        test('Test some resolved deploy blockers', () => {
+            githubUtils.generateStagingDeployCashBody(tag, basePRList, basePRList, baseDeployBlockerList, [baseDeployBlockerList[0]])
+                .then((issueBody) => {
+                    // eslint-disable-next-line max-len
+                    expect(issueBody).toBe(`${allVerifiedExpectedOutput}${deployBlockerHeader}${closedCheckbox} ${baseDeployBlockerList[0]}\r\n${openCheckbox} ${baseDeployBlockerList[1]}\r\n`);
+                });
+        });
+
+        test('Test all resolved deploy blockers', () => {
+            githubUtils.generateStagingDeployCashBody(tag, basePRList, basePRList, baseDeployBlockerList, baseDeployBlockerList)
+                .then((issueBody) => {
+                    // eslint-disable-next-line max-len
+                    expect(issueBody).toBe(`${allVerifiedExpectedOutput}${deployBlockerHeader}${closedCheckbox} ${baseDeployBlockerList[0]}\r\n${closedCheckbox} ${baseDeployBlockerList[1]}\r\n`)
+                });
+        });
+    })
```

Comment:
Is there a way to test that the output of `createNewStagingDeployCash` equals the [body here](https://github.com/Expensify/expensify-common/pull/347/files#diff-27894ed198bf5b3cf8699ff675f2ad04f08c0cdc89623579c98431b8c5c91627R21)? Basically let's make sure they will play well together, not just in isolation.

Reply:
Added a test